watertap/unit_models/tests/test_electrodialysis_0D.py

```diff
@@ -1133,7 +1134,7 @@ def test_initialization_scaling(self, EDcell):
         iscale.set_scaling_factor(m.fs.unit.cell_length, 10)
         iscale.set_scaling_factor(m.fs.unit.cell_pair_num, 0.1)
         iscale.calculate_scaling_factors(m.fs)
-        initialization_tester(m)
+        initialization_tester(m, outlvl=idaeslog.DEBUG)
```

Comment:
Is this change just lingering from actually debugging the test file? Probably shouldn't remain.

Reply:
I honestly don't fully know the implication of different `outlvl` setting.  We can change it back if the change is not proper.